crates/binding_core_wasm/src/lib.rs

```diff
@@ -75,7 +75,7 @@ pub fn minify_sync(s: JsString, opts: JsValue) -> Result<JsValue, JsValue> {
 
     try_with_handler(
         c.cm.clone(),
-        swc::HandlerOpts {
+        swc_core::base::HandlerOpts {
```

Comment:
I think `swc_error_reporters` deserves a own module, because it can be useful for someone does not need full swc

Reply:
Yes, maybe I came to have some similar opinion around other dependency reorg. For now, pr focused to move things transparently first and then apply necessary changes on top of that. I'll note this as well.